Editor/Resources/EditorWindow/GameLiftPlugin.uxml

```diff
@@ -0,0 +1,37 @@
+<!-- Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. -->
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+
+<ui:UXML xmlns:ui="UnityEngine.UIElements" xmlns:uie="UnityEditor.UIElements" xsi="http://www.w3.org/2001/XMLSchema-instance" engine="UnityEngine.UIElements" editor="UnityEditor.UIElements" noNamespaceSchemaLocation="../../../../../UIElementsSchema/UIElements.xsd" editor-extension-mode="False">
+    <Style src="./common.uss" />
```

Comment:
Is there a way to import this in the `.uss` like is done for `.css`? It'll make it clearer where the variables come from at least

Reply:
I wasnt sure if `@import` is supported as its not documented in their docs, but it does seem to work in practice! :)